tests/test_user_login_unit.py

```diff
@@ -0,0 +1,32 @@
+"""
+Unit tests for user authentication utility
+"""
+
+def check_password_changed(user, new_pass) -> bool:
+    return user.check_password(new_pass)
+
+class TestChangePasswordUnit:
+    def test_change_password_success(self, unregistered_user):
```

Comment:
Should have type hints for `unregistered_user` (check the type returned in the fixture if you dont know the return type)